src/Utilities.cpp

```diff
@@ -707,3 +707,18 @@ OsmAnd::PointD OsmAnd::Utilities::getTileEllipsoidNumberAndOffsetY(int zoom, dou
     res.y = floor(((xTilesCountForThisZoom / 2 - M2 * xTilesCountForThisZoom / 2 / M_PI) - yTileNumber) * tileSize);
     return res;
 }
+
+std::pair<int, int> OsmAnd::Utilities::calculateFinalXYFromBaseAndPrecisionXY(int bazeZoom, int finalZoom, int precisionXY,
+                                                                              int xBase, int yBase, bool ignoreNotEnoughPrecision) {
```

Comment:
Please format the code according to C formatting style (curly brace is placed on a separate line) here and everywhere else throughout the commit.